src/neural/blas/network_blas.cc

```diff
@@ -379,6 +384,11 @@ BlasNetwork::BlasNetwork(const WeightsFile& file, const OptionsDict& options)
                                                        pol_channels, channels);
   }
 
+#ifdef USE_EIGEN
+  std::cerr << "Using Eigen version " << EIGEN_WORLD_VERSION << "."
```

Comment:
Use `CERR` from logging.h instead so that it also gets into log?

Reply:
Should I do this for the other instances in the file as well, or save that for a separate pr?

Reply:
Either is fine, probably better to keep changes separate.